stable/chat/v1/messages.proto

```diff
@@ -9,53 +9,55 @@ option go_package = "github.com/harmony-development/legato/gen/chat/v1";
 
 // Overrides provide a way to override the name and avatar of a message.
 message Overrides {
-  // the overridden username.
+  // The overridden username.
   optional string username = 1;
-  // the overridden avatar.
+  // The overridden avatar.
+  //
+  // This can be a HMC, a file ID, or an external image URL.
```

Comment:
it may be worth considering if HMC is even necessary if external image URLs are supported here. The path for fetching media is stable and well defined.

Reply:
It's not necessary, it's just a convenience for clients, so that they don't have to convert it to a proper URL with path